apps/hub/src/components/login-form.tsx

```
@@ -0,0 +1,68 @@
+import { Button } from '@cove/ui/components/button'
+import { Input } from '@cove/ui/components/input'
+import { Label } from '@cove/ui/components/label'
+import { cn } from '@cove/ui/lib/utils'
+
+export function LoginForm({ className, ...props }: React.ComponentProps<'div'>) {
+  return (
+    <div className={cn('flex flex-col gap-6', className)} {...props}>
+      <form>
+        <div className="flex flex-col gap-6">
+          <div className="flex flex-col items-center gap-2">
+            <a href="#" className="flex flex-col items-center gap-2 font-medium">
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)